scripts/augmented_line_connections.py

```diff
@@ -169,8 +171,5 @@ def haversine(p):
             lifetime=costs.at["HVAC overhead", "lifetime"],
         )
 
-        _set_dc_underwater_fraction(n.links, snakemake.input.regions_offshore)
-        _set_dc_underwater_fraction(n.lines, snakemake.input.regions_offshore)
```

Comment:
Cool :)
May you please comment these lines instead of deleting them and moving the TODO, right before these two (future) commented lines?
Ready to merge thereafter :)

Reply:
Done! :)